metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -1179,7 +1187,12 @@ def get_tables_for_dataset(
                             sharded_tables[table_name] = table
                         continue
                 else:
```

Comment:
This `else` statement looks unnecessary because we always `continue` `if shard`. Not a big deal, but I had some trouble following that `table_items` is always updated if there's no shard